rasa_nlu/extractors/crf_entity_extractor.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import print_function
 from __future__ import division
 from __future__ import absolute_import
+from  future.utils import PY3
```

Comment:
No. Forget to remove it. I was about to make some separate serialization there: crf_entity_extractor relies on joblib.dumps() which use different protocol depending on the Python version making a model trained with Rasa and python 2 incompatible with Rasa and python 3 and vice-versa.